### PR TITLE
Changed "ukw100/IRMP" to "IRMP-org/IRMP", because it is permanently moved anyway

### PR DESCRIPTION
**Pull request by ArminJo**

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @ArminJo. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/11/12 15:47:51 Loaded 7456 libraries from DB
Changing URL of library IRMP from https://github.com/ukw100/IRMP.git to https://github.com/IRMP-org/IRMP.git
Success!
```

**Pull request merged**
